TrackierSDK/sdk-core/src/main/java/com/trackier/sdk/Util.kt

```diff
@@ -59,6 +59,13 @@ object Util {
 
         return printHexBinary(sha256Hmac.doFinal(data.toByteArray()))
     }
+    
+    fun retargettingData(url: String, time: String): String {
+        val body = mutableMapOf<String, Any>()
+        body["date&time"] = time
```

Comment:
key name should not have &

```suggestion
        body["rtgtime"] = time
```

Reply:
fixed